src/pages/404.tsx

```diff
@@ -1,14 +1,12 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
Please fix the error, one way to achieve it is that you can return an empty div, instead of null in the component.
I guess that if you unconditionally return null, then eslint may not recognize it as a component.